File: 02_Lenguajes-de-programación/02_Java/07_clases.java
package modulo;

public class Lenguaje {
    String nombre;
    Integer anio;

    public Lenguaje() {}

    public Lenguaje(String nombre, Integer anio) {
        this.nombre = nombre;
        this.anio = anio;
    }

    public void descripcion() {
        System.out.println(this.nombre + " fue creado en " + this.anio);
    }
}

File: 02_Lenguajes-de-programación/03_C#/07_clases.cs
using System;

public class Lenguaje
{
    private string nombre;
    private int anio;
    public Lenguaje(string nombre, int anio)
    {
        this.nombre = nombre;
        this.anio = anio;
    }

    public void descripcion()
    {
        Console.WriteLine("{0} fue creado en {1}", this.nombre, this.anio);
    }
}

class Program {
  public static void Main (string[] args) {

    Lenguaje html = new Lenguaje("HTML", 1993);
    html.descripcion();

    Lenguaje css = new Lenguaje("CSS", 1996);
    css.descripcion();

    Lenguaje javascript = new Lenguaje("JavaScript", 1995);
    javascript.descripcion();
  }
}
